packages/mobile/src/screens/edit-track-screen/screens/ListSelectionScreen.tsx

```diff
@@ -68,7 +74,9 @@ export const ListSelectionScreen = (props: ListSelectionProps) => {
     onChange,
     value,
     searchText = '',
-    disableSearch = false
+    disableSearch = false,
+    disableReset = false,
```

Comment:
nit: took me a second to understand what this did - maybe rename to something like `allowDeselect` (and invert logic)?

Reply:
agreed